app/client/reducers/creditTransfer/actions.ts

```diff
@@ -0,0 +1,67 @@
+import { createAction, ActionsUnion } from "../../reduxUtils";
+import {
+  CreditTransfers,
+  CreditTransferActionTypes,
+  LoadCreditTransferActionTypes,
+  ModifyCreditTransferActionTypes,
+  LoadCreditTransferPayload,
+  ModifyCreditTransferRequestPayload,
+  CreateCreditTransferPayload,
+  ModifyCreditTransferPayload
+} from "./types";
+
+export const LoadCreditTransferAction = {
+  loadCreditTransferListRequest: (payload: LoadCreditTransferPayload) =>
+    createAction(
+      LoadCreditTransferActionTypes.LOAD_CREDIT_TRANSFER_LIST_REQUEST,
+      payload
+    ),
+  loadCreditTransferListSuccess: () =>
+    createAction(
+      LoadCreditTransferActionTypes.LOAD_CREDIT_TRANSFER_LIST_SUCCESS
+    ),
+  loadCreditTransferListFailure: (err: string) =>
+    createAction(
+      LoadCreditTransferActionTypes.LOAD_CREDIT_TRANSFER_LIST_FAILURE,
+      err
+    )
+};
```

Comment:
@gtaschuk I refactored to using the `createAction` generic. Can you please do the same in https://github.com/teamsempo/SempoBlockchain/pull/298 for readability/maintainability